tests/unit/FormTextArea.spec.js

```diff
@@ -89,9 +92,18 @@ describe('FormTextArea', () => {
     expect(wrapper.find('.invalid-feedback').exists()).toBe(true);
     expect(wrapper.find('.invalid-feedback').isVisible()).toBe(true);
     expect(wrapper.find('.invalid-feedback').text()).toContain(errorText);
-    expect(wrapper.find('.invalid-feedback').text()).toContain(requiredText);
     expect(wrapper.find('textarea').classes('is-invalid')).toBe(true);
+    expect(wrapper.find('textarea').element.rows).toBe(4);
```

Comment:
What value is this assertion adding? If the number of rows happens be be greater or less I think it would still be fine.